packages/code-du-travail-ui/src/layout/Container/index.js

```diff
@@ -28,11 +34,20 @@ const StyledContainer = styled.div`
   max-width: ${breakpoints.desktop};
   margin: 0 auto;
   padding: 0 ${spacing.medium};
-  ${props =>
-    props.narrow &&
-    css`
-      max-width: 43.75rem; //700px;
-    `};
+  ${props => {
+    if (props.narrow) {
+      const maxWidth = "46.25rem"; //740px
+      if (props.noPadding) {
+        return css`
+          padding: 0;
+          max-width: ${maxWidth};
+        `;
+      }
+      return css`
+        max-width: ${maxWidth};
```

Comment:
can we have no-padding without narrow ?

Reply:
The purpose of the container is to add paddings if not used with `narrow`